app/main.py

```diff
@@ -1,25 +1,65 @@
-from abc import ABC
+class IntegerRange:
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
 
+    def __set_name__(
+            self, owner: "SlideLimitationValidator",
+            name: str) -> None:
+        self.protected_name = f"_{name}"
 
-class IntegerRange:
-    pass
+    def __get__(
+            self, instance: "SlideLimitationValidator",
+            owner: "SlideLimitationValidator") -> str:
+        return getattr(instance, self.protected_name)
+
+    def __set__(
+            self, instance: "SlideLimitationValidator",
+            value: int) -> None:
+        if not isinstance(value, int):
+            raise TypeError("value should be int")
+        elif not self.min_amount <= value <= self.max_amount:
+            raise ValueError("value is out of range")
+        setattr(instance, self.protected_name, value)
 
 
 class Visitor:
-   pass
+    def __init__(self, name: str, age: int, weight: int, height: int) -> None:
+        self.name = name
+        self.age = age
+        self.weight = weight
+        self.height = height
 
 
-class SlideLimitationValidator(ABC):
-    pass
+class SlideLimitationValidator:
```

Comment:
Keep `SlideLimitationValidator` class inheritance from the `ABC`, do not remove it